eng/pipelines/coreclr/templates/run-superpmi-replay-job.yml

```diff
@@ -65,9 +65,16 @@ jobs:
     steps:
     - ${{ parameters.steps }}
 
-    - script: |
-        mkdir -p $(SpmiCollectionLocation)
-      displayName: Create directory for SPMI collection
+    - ${{ if ne(parameters.osGroup, 'windows') }}:
```

Comment:
`mkdir -p foo` on windows creates a `-p` directory and a `foo` directory.

We could remove the `osGroup` checking, but I figured we might as well be consistent everywhere and check it just in case we expand to support running on non-Windows OS? I was trying to make the various YML files consistent.

Reply:
My take is we can add the various `osGroup` checks in future when we make this compatible for non-Windows OS. For now, it just adds maintenance overhead so I would just remove everything for non-Windows.

Reply:
Another reason I don't think we should include unnecessary things in these yml is mostly we end up copying them to create new pipeline and the irrelevant things get duplicated in new files as well.